main.tf

```diff
@@ -111,6 +111,10 @@ resource "azurerm_postgresql_flexible_server" "main" {
     }
   }
   depends_on = [azurerm_private_dns_zone_virtual_network_link.main, azurerm_private_dns_zone_virtual_network_link.main2]
+  
+  lifecycle {
+    ignore_changes = [high_availability.0.standby_availability_zone]
```

Comment:
Thanks for you changes but here is one tf-lint regarding , Please fix this so I can merge this PR

Reply:
@d4kverma Thanks for the quick response I did a fix but I see one check is still failing regarding GitHub Secret.

Reply:
because you are an authorised person. that is fine. 
thanks for your contribution.

Reply:
I still see checks are failing